.changeset/green-papayas-thank.md

```diff
@@ -0,0 +1,7 @@
+---
+'@rocket.chat/model-typings': patch
+'@rocket.chat/core-typings': patch
+'@rocket.chat/meteor': patch
+---
+
+Fixed: 🛠️ Resolved issues with updating custom emojis, including file upload and image caching problems.
```

Comment:
Please, follow the patterns described in a section of this page: https://developer.rocket.chat/docs/development-workflow